README.md

```diff
@@ -14,96 +14,135 @@ external dependencies, editor integration, and quick feedback.
 
```

Comment:
In the tutorial I had this paragraph:

```
This gives you `nix` editor integration for free. Whenever you open
a file, as long as there is a `direnv`/`lorri` setup in the project,
your editor automatically loads all tools, environment variables,
library dependencies, etc. from your nix files. In short: Everything
you need to be productive on your project and everything to make sure
your codevelopers have the same, reproducible setup.
Even further, if you change something in your nix files, `lorri`
immediately picks it up, and your editor does as well.
```

Can we salvage some of that?

Reply:
I use a pretty vanilla Vim setup for most of my coding.

> your editor automatically loads all tools, environment variables,
library dependencies, etc. from your nix files.

This just isn't really true for example for my development experience. Whether I'm inside or outside a `direnv` directory has no impact on my editor at all, except that other command line tools are available via `:! <tool>`.

*However*, I do think that users (including me) can probably benefit from learning about how to integrate their editor(s) with `direnv` for a better development experience. This is why I wanted to keep the "How to integrate with Emacs" tutorial (tentatively in `contrib/EMACS.md`), and I think there should be a prominent link to it from the top-level README (see https://github.com/target/lorri/pull/233#issuecomment-556054792).

Reply:
There's a chance that because I haven't made an effort to customise my development experience based on per-project `direnv` / `lorri` things, I'm underestimating the importance of the customisation possibilities that come with the `direnv` / `lorri` duo.

If you would like the messaging to go more in this direction, I'm happy to oblige - in that case, there should be at least a teaser about editor customisation directly in the top-level README.

Reply:
> This just isn't really true for example for my development experience. Whether I'm inside or outside a `direnv` directory has no impact on my editor at all, except that other command line tools are available via `:! <tool>`.

You are missing out very much. Good editor tooling support is a base premise of lorri and improves productivity significantly in my experience.

The paragraph was from back when it was Emacs-specific, but we should strongly recommend users to set up editor tooling, because lorri makes it often possible in the first place.

> I'm underestimating the importance of the customisation possibilities that come with the `direnv` / `lorri` duo.

yeah! try it out :)

Reply:
I can show you some of the stuff I use if you want.

Reply:
I added a stub "Editor integration" section with a link to `contrib/emacs.md` and #244 as a follow-up to flesh this out more.